gradle.properties

```diff
@@ -4,6 +4,25 @@ kotlin.code.style=official
 kotlin.js.generate.executable.default=false
 android.useAndroidX=true
 
+# Required to publish to Nexus (see https://github.com/gradle/gradle/issues/11308)
+systemProp.org.gradle.internal.publish.checksums.insecure=true
+
+GROUP=com.squareup.logcat
+VERSION_NAME=1.0-SNAPSHOT
+
+POM_DESCRIPTION=A tiny Kotlin API for cheap logging on top of Android's normal Log class.
+
+POM_URL=https://github.com/square/logcat/
+POM_SCM_URL=https://github.com/square/logcat/
+POM_SCM_CONNECTION=scm:git:git://github.com/square/logcat.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/square/logcat.git
+
+POM_LICENCE_NAME=The Apache Software License, Version 2.0
+POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
+POM_LICENCE_DIST=repo
+
+POM_DEVELOPER_ID=square
+POM_DEVELOPER_NAME=Square, Inc.
```

Comment:
I'm a bit surprised you had to add all this, we've published releases just fine in the past, what's changed?

Reply:
I used a skeleton KMP project to help get the Android source setup with the build files - this must have come over with it.  I'll remove these extra lines.

I have no experience publishing an AAR, so I'll have to dig in to learn about the publishing process.

Reply:
These have been removed and I was able to run the Android build and sample project on the emulator.  The publishing plugin was re-added when I restructured in [3c402d1](https://github.com/square/logcat/pull/15/commits/3c402d1f837a1a1b50bf2719ee05a908ed38b67c) so I think the AAR publishing should be OK now